example/javalib/linting/3-palantirjavaformat/src/A.java

```diff
@@ -0,0 +1,84 @@
+package com.google.googlejavaformat.java.test;
```

Comment:
For the example, let's make it small (e.g. ~5 lines), and include it in the example page via a `/** See Also: src/A.java */` comment. That would help make the example more complete and understandable to readers, while the unit tests cover the large input scenarios for us.

Reply:
Updated. Please review.